marytts-languages/marytts-lang-en/src/main/java/marytts/language/en/Preprocess.java

```diff
@@ -153,6 +153,7 @@
 		contractions.put("'d", new String[] { "d" });
 		contractions.put("'m", new String[] { "m" });
 		contractions.put("'re", new String[] { "r" });
+		contractions.put("'t", new String[] {"t"});
```

Comment:
>  isn't, wasn't, can't, and so on.

All of the examples [you mention](https://github.com/marytts/marytts/pull/1003#issue-428859932) seem to also include an *n* before the apostrophe.

```suggestion
		contractions.put("n't", new String[] {"t"});
```
Wouldn't this work and capture these examples (and others)?

Reply:
I don't think it makes a difference.  However, as as my further comments below suggest, I think the code for handling contractions is currently incomplete.